kore/src/main/java/org/kframework/utils/errorsystem/KException.java

```diff
@@ -138,6 +138,7 @@ public enum ExceptionType {
     IGNORED_ATTRIBUTE,
     REMOVED_ANYWHERE,
     DEPRECATED_DIRECTORY_FLAG,
+    DEPRECATED_SYMBOL,
```

Comment:
What's the rationale for making a new error instead of using `FUTURE_ERROR`?

Reply:
I guess my reading is that `FUTURE_ERROR` means that a decision made by the _compiler_ will break this in the future, whereas we're saying here that it's a decision made by the _semantics_ developer. It's a pretty thin line though, I agree

Reply:
That makes sense, I see the distinction. Maybe more broadly I'm unclear on what the use case for these ExceptionTypes is, `-W` options? Message templates? Just an idle pondering. Maybe in the future we can visit this.